### ISSUE TITLE
React.createFactory() is deprecated

### ISSUE THREAD

**Issue by styfle** [styfle/react-server-example-tsx#280]
Need to figure out a different solution

**Comment by mullinsmikey** [comment]
Hi @styfle 

Consider this solution:
`const stream = renderToNodeStream(createElement(App, fetchProps()));`

**Comment by styfle** [comment]
Yep, or I could change ts to tsx and solve it like this https://github.com/styfle/packagephobia/pull/534

**Comment by mullinsmikey** [comment]
Oh my, I've already changed this boilerplate a lot. Used it as a learn-it-yourself tutorial and it works fast as hell.
